mindjunkies/courses/views.py

```
@@ -9,9 +9,10 @@
 from django.views.decorators.http import require_http_methods
 from django.views.generic import TemplateView
 from django.views.generic.edit import CreateView, UpdateView
+from django.views.generic import TemplateView
 
 from mindjunkies.courses.models import Course, CourseCategory, Enrollment
-
+from .models import Course, Enrollment
```

Ruff (0.8.2)

Redefinition of unused `Course` from line 14

(F811)